projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -507,6 +507,16 @@ export interface FeatureTogglesInterface {
    * customization buttons in the BottomHeaderSlot in SmartEdit.
    */
   cmsBottomHeaderSlotUsingFlexStyles?: boolean;
+
+  /**
+   * 1. It uses the new `SiteThemeService` as the source of truth for the "site theme" value
+   * (this value can change over time, e.g. when selecting new value in the new `SiteThemeSwitcherComponent`).
+   * Previously the "site theme" could be set only on the page start (via the static config `config.context.theme` or via CMS, when using the feature of the "automatic site-context configuration").
+   * 2. Now, when no custom theme is selected, the default theme value is an empty string `''`,
+   * unless you configure it differently via the global config `config.context.theme` (or via CMS).
+   * Previously, there the non-defined theme had a value `undefined`.
+   */
+  useNewSiteThemeSwitcher?: boolean;
```

Comment:
Regarding comment from @Platonn: https://github.com/SAP/spartacus/pull/19016#discussion_r1752088208
I've noticed the JSDoc is adjusted, but the feature toggle name has not changed. I'd opt for changing the name to the proposed one.

Reply:
I missed.. i just updated